### PR TITLE
Nextcloud Integration

### PR DESCRIPTION
**Pull request by meichthys**
**Description:**
This pull request provides documentation for the new nextcloud integration.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30871

## Checklist:

- [ x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/nextcloud.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  type: string
+username:
+  description: The username of a nextcloud user that has access to the nextcloud monitor api.
+  required: true
+  type: string
+password:
+  description: The app password generated from the Nextcloud security settings page.
+  required: true
+  type: string
+scan_interval:
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]: Please remove this, as this is provided by the platform schema.

File: source/_integrations/nextcloud.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ha_release: 0.104
+---
+
+The `nextcloud` platform pulls summary [Nextcloud](https://nextcloud.com/) information into home-assistant.
+
+![Nextcloud Example Sensor]('/images/screenshots/nextcloud-sample-sensor.png')
+
+## Configuration
+
+This integration requires access to the monitor api of a Nextcloud instance (This is generally an admin user).
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]:
```suggestion
This integration requires access to the monitor API of a Nextcloud instance (This is generally an admin user).
```

File: source/_integrations/nextcloud.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+You should also generate an App password from the nextcloud web ui. `Settings > Security > Devices & sessions > Create new app password`
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]:
```suggestion
You should also generate an App password from the Nextcloud web UI. `Settings > Security > Devices & sessions > Create new app password`.
```

File: source/_integrations/nextcloud.markdown [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  - platform: nextcloud
+    url: Https://YOUR_NEXTCLOUD_URL
+    username: YOUR_USERNAME
+    password: YOUR_APP_PASSWORD
+
+```
+
+{% configuration %}
+url:
+  description: The full url to your nextcloud instance.
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]:
```suggestion
  description: The full URL to your Nextcloud instance.
```

File: source/_integrations/nextcloud.markdown [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  required: true
+  type: string
+username:
+  description: The username of a nextcloud user that has access to the nextcloud monitor api.
```
Comment (frenck) [inline source/_integrations/nextcloud.markdown]:
```suggestion
  description: The username of a Nextcloud user that has access to the Nextcloud monitor API.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @meichthys!

I've reviewed it, and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/nextcloud.markdown (hunk 1 above)
Comment (meichthys) [reply source/_integrations/nextcloud.markdown]: @frenck Thanks for the review m8!

I'm a little confused. This suggestion seems to indicate that i should remove the {% configuration %} section, but then the other suggestions are indicating that i should make changes to the configuration section. Is there just a specific part of the configuration section that i should remove?

Comment (frenck) [reply source/_integrations/nextcloud.markdown]: The display is a bit weird in GitHub, odd 😟 
You did change it right though 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/nextcloud.markdown [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+title: Nextcloud Sensor
+description: Instructions on how to integrate Nextcloud monitor api data into home-assistant.
+logo: nextcloud.png
```
Comment (klaasnicolaas) [inline source/_integrations/nextcloud.markdown:4]: Missing the logo file

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/nextcloud.markdown (hunk 6 above)
Comment (meichthys) [reply source/_integrations/nextcloud.markdown:4]: @klaasnicolaas Thanks! I've added the logo file - I'm not sure if it needs to be resized or not.

**Comment by meichthys** [comment]
Thanks @frenck !

**Pull request merged**